app/controllers/accountants_controller.rb

```diff
@@ -38,15 +45,21 @@ def edit
 
   private
 
+  def generate_base_patients
+    patients = pledged_patients.includes(:clinic)
+    @patients = paginate_results(patients)
+
+    @entry_name = t('common.patient')
+  end
+
   def paginate_results(results)
     Kaminari.paginate_array(results)
             .page(params[:page])
             .per(25)
   end
 
   def pledged_patients
-    Patient.where(pledge_sent: true,
-                  initial_call_date: 6.months.ago..)
```

Comment:
Just to telegraph, I think I'm good taking out this filter; Kaminari prevents us from loading a million billion patients all at once, which is what we were going for with this original rule.

Reply:
to clarify, do you want to remove it entirely? and are you referring to both `pledge sent` and `initial call date` or just the latter?

Reply:
The way you have it is right - we def want the `pledge_sent` (since that's what the accountants/reconcilers care about) but don't need the `initial_call_date` because of kaminari.